agent/test/oome/testcase.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+### Usage
+###   - Without Valgrind
+###       ./testcase.sh /path/to/heapstats
+###
+###   - With Valgrind
+###       VALGRIND_OPTION="--leak-check=yes --suppressions=/path/to/suppress" \
```

Comment:
It seems to me that we should add an argument for testcase.sh to run with valgrind easy instead of VALGRIND_OPTION.
ex) ./testcase.sh /path/to/heapstats /path/to/supress
then, we can forget "--leak-check=yes" and easy to add other suppress without editing "VALGRIND_OPTION".